README.md

```diff
@@ -77,6 +79,56 @@ git submodule add https://github.com/greenfork/jzignet libpath
 const jzignet = @import("jzignet");
 ```
 
+## How to include as dependency (with zon)
+
+This method of including jzignet in your project is more complicated.
+
+First, create a file named `build.zig.zon` with the following content. Replace `<commit>` in the file with the git commit you want to use, for example, `943ace7`.
+
+```zig
+.{
+    .name = "hello-world",
+    .version = "0.0.1",
+    .dependencies = .{
+        .jzignet = .{
+	        .url = "https://github.com/greenfork/jzignet/archive/<commit>.tar.gz",
```

Comment:
The keybase issue says that GitHub can arbitrarily change archived source tarballs. We should probably use tags here and manually upload the archives?

Reply:
Not it is not a problem. andrewk said that the Zig hash won't change, since it's based on extracted content.

Also, btrfs on Linux has a bug that made the hash different, which is fixed just now (Linux 6.5.3).

Reply:
> Don't merge this yet! You need to publish your own archive files, and change this patch to mention the files you created.

What did you mean by this then? GitHub already has archives for each release

Reply:
I misunderstood how zig's hash works. Apparently, you can use the Github tarball.